src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/PriceControls.tsx

```diff
@@ -113,16 +118,19 @@ const PriceControls: React.FC<Props> = ({
         spread={spreadValue}
         show={showPriceMovement}
         isAnalyticsView={isAnalyticsView}
+        isRequestRFQ={Boolean(isRfqStateCanRequest || isRfqStateRequested)}
       />
-      <TileBookingSwitch
-        isTradeExecutionInFlight={isTradeExecutionInFlight}
-        currencyPair={currencyPair}
-        notional={notional}
-        rfq={rfq}
-        rfqState={rfqState}
-        hasUserError={hasUserError}
-        isAnalyticsView={isAnalyticsView}
-      />
+      {lastTradeExecutionStatus ? null : (
```

Comment:
A nice little trick to avoid the ternary: 

```typescript
{lastTradeExecutionStatus || (
  <TileBookingSwitch
    isTradeExecutionInFlight={isTradeExecutionInFlight}
    currencyPair={currencyPair}
    notional={notional}
    rfq={rfq}
    rfqState={rfqState}
    hasUserError={hasUserError}
    isAnalyticsView={isAnalyticsView}
  />
)}

```